com.unity.render-pipelines.high-definition/Runtime/PostProcessing/PostProcessSystem.cs

```diff
@@ -1585,9 +1611,12 @@ RTHandle Allocator(string id, int frameIndex, RTHandleSystem rtHandleSystem)
                 );
             }
 
-            next = camera.GetCurrentFrameRT((int)HDCameraFrameHistoryType.TAAMotionVectorMagnitude)
-                ?? camera.AllocHistoryFrameRT((int)HDCameraFrameHistoryType.TAAMotionVectorMagnitude, Allocator, 2);
-            previous = camera.GetPreviousFrameRT((int)HDCameraFrameHistoryType.TAAMotionVectorMagnitude);
+            int historyType = (int) (postDoF ?
+                HDCameraFrameHistoryType.TAAMotionVectorMagnitude : HDCameraFrameHistoryType.TAAMotionVectorMagnitudePostDoF);
```

Comment:
You don't need the post dof as we don't do speed rejection in the ad-hoc pass. No need to allocate another history buffer

Reply:
In fact we don't need to modify this function at all, just don't call it when you do your ad-hoc TAA pass

Reply:
Good to know, will push the change (along with the other comments) soon.